code/controllers/subsystem/autotransfer.dm

```diff
@@ -19,3 +21,14 @@ SUBSYSTEM_DEF(autotransfer)
 	if(world.time > targettime)
 		SSvote.initiate_vote("transfer", null)
 		targettime = targettime + CONFIG_GET(number/vote_autotransfer_interval)
+
+/datum/controller/subsystem/autotransfer/proc/try_vote(mob/user, vote_name)
+	if(!CONFIG_GET(flag/head_transfer_vote))
+		return
+	if(!SSshuttle.canEvac(user))
+		return
+	if(world.time < cooldown_time)
+		to_chat(user, "<span class='warning'>A transfer vote has been triggered recently. Please wait [DisplayTimeText(cooldown_time - world.time)] or contact your station's captain for emergency evacuation.</span>")
```

Comment:
I'm assuming that one of the things that spurred this on is "I'm bored calling shuttle" situations, but anecdotally it's usually the captain doing that. 

I also don't know why power needs to be taken away from heads when SOP could cover this (I believe it... implies it? but is not explicit)

Otherwise I think the concept of being able to initialize a transfer vote is... intriguing, but idk how players would use it. Maybe they'll do it when they realize it's a greenshift which will irk some players, but be good for others.